src/org/loklak/harvester/TwitterScraper.java

```diff
@@ -416,9 +416,10 @@ public String toString() {
     
     public static class TwitterTweet extends MessageEntry implements Runnable {
 
-        public final Semaphore ready;
-        public UserEntry user;
-        public boolean writeToIndex, writeToBackend;
+        private final Semaphore ready;
+        private UserEntry user;
+        public boolean writeToIndex;
+        public boolean writeToBackend;

```

Comment:
@vibhcool Instead of using `getter` methods please use `Reflection`. For more please look into the SO link, [testing a class having private fields and methods](http://stackoverflow.com/questions/34571/how-do-i-test-a-class-that-has-private-methods-fields-or-inner-classes)